packages/components/src/components/as-stacked-bar-widget/README.md

```diff
@@ -154,3 +154,46 @@ lang: javascript
 ---
 stackedBar.showLegend = 'false';
 ```
+
+#### **isLoading**: boolean=false
+Indicates if the widget is loading or not. Defaults to false.
```

Comment:
Same description as above.

Reply:
I dont get you :S

Reply:
I think he means put the same one he mentioned above, so

`* Boolean property to control the widget loading state. If true, a spinner is shown.`